app/Http/Controllers/Auth/AuthController.php

```diff
@@ -38,40 +36,38 @@ public function getCheckpoint()
 
     public function getIndex()
     {
+        $state = str_random(5);
```

Comment:
Why not just use a UUID if you need uniqueness and avoid the headache of guessing if it'll be unique or not?

Also, reviewboard is that way -> http://reviewboard.freedom.tm/